ngraph/python/src/pyngraph/node.cpp

```diff
@@ -87,6 +87,7 @@ void regclass_pyngraph_Node(py::module m)
     node.def("outputs",
              (std::vector<ngraph::Output<ngraph::Node>>(ngraph::Node::*)()) &
                  ngraph::Node::outputs);
+    node.def("is_same_op_type", &ngraph::Node::is_same_op_type);
```

Comment:
@ilyachur do we need this method in Node class? I think we do not use it anywhere

Reply:
No we don't need this method. I have a plan to remove it.

Reply:
I only use it as matcher of two nodes in helper function.

Reply:
But at the current moment as I can see we don't use this method. Please correct me if i am not right.